src/Console/Command/FixCommand.php

```diff
@@ -292,8 +294,8 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $stdErr = ($output instanceof ConsoleOutputInterface) ? $output->getErrorOutput() : null;
-        if ($stdErr && extension_loaded('xdebug')) {
+        $stdErr = ($output instanceof ConsoleOutputInterface) ? $output->getErrorOutput() : $output;
+        if (extension_loaded('xdebug')) {
```

Comment:
it is printen to `stdErr` (line 297), I'm not sure I follow. If there is no 2 stream support by the console, it will print to the same, the other option would be to not print the info at all... not sure what is better